topics/intro/content_updates.md

```diff
@@ -12,6 +12,11 @@ See [GitHub Changelog](https://github.com/JetBrains/intellij-sdk-docs/commits/ma
 
 ## 2022
 
+### May-22
+
+Inlay Hints
+: Add [](inlay_hints.md) section explaining how to implement editor inlay hints.
```

Comment:
remove "section". it would be nice if we can add concise explanation of what they are used for in short sentence

Reply:
The word "section" is fine and we used it already 13 times but I added a short explanation of what inlay hints are.